internal/templates/template.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package templates
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.jetpack.io/devbox/internal/boxcli/usererr"
+)
+
+func Init(w io.Writer, template, dir string) error {
+	if err := createDirAndEnsureEmpty(dir); err != nil {
+		return err
+	}
+
+	templatePath, ok := templates[template]
+	if !ok {
+		return usererr.New("unknown template %q", template)
+	}
+
+	tmp, err := os.MkdirTemp("", "devbox-template")
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	cmd := exec.Command(
+		"git", "clone", "git@github.com:jetpack-io/devbox.git", tmp,
+	)
+	fmt.Fprintf(w, "%s\n", cmd)
```

Comment:
remove?

Reply:
I like seeing this.  (it was for actual output not for debug purposes). Do you feel strongly?

Reply:
oh no worries, I thought it was debug code.